### PR TITLE
Fix: remove server side apply duplicated sentence

### PR DESCRIPTION
**Pull request by zhangzhiqiangcs**
Remove the duplicated sentence as it has appeared in pre paragraph.

### CODE REVIEW

File: content/en/docs/reference/using-api/server-side-apply.md [hunk 1]
```diff
@@ -561,14 +561,6 @@ Downgrading works because kubectl Server-Side Apply keeps the
...
 `kubectl apply`.
 
-This behavior applies to Server-Side Apply with the `kubectl` field manager.
-As an exception, you can opt-out of this behavior by specifying a different,
-non-default field manager, as seen in the following example. The default field
-manager for kubectl server-side apply is `kubectl`.
-
-```shell
-kubectl apply --server-side --field-manager=my-manager [--dry-run=server]
-```
```
Comment (tengqm) [inline content/en/docs/reference/using-api/server-side-apply.md:571]: Is this repeating text intentional?

Comment (zhangzhiqiangcs) [reply content/en/docs/reference/using-api/server-side-apply.md:571]: I guess it's not. It's talking about upgrade from client-side-apply to server-side-apply, it belongs to pre sections.

Comment (zhangzhiqiangcs) [reply content/en/docs/reference/using-api/server-side-apply.md:571]: @tengqm Any suggestions? I not very sure about this.

Comment (tengqm) [reply content/en/docs/reference/using-api/server-side-apply.md:571]: I'm not that familiar with SSA either. Sorry.

Comment (kbhawkey) [reply content/en/docs/reference/using-api/server-side-apply.md:571]: The original content is not entirely clear. Best to ask for a technical review.

Comment (zhangzhiqiangcs) [reply content/en/docs/reference/using-api/server-side-apply.md:571]: Sure, but how can I request for a technical review? @kbhawkey

**Comment by tengqm** [comment]
I don't think this is a simple repeat.
/close

**Pull request closed without merging**